internal/cmd/operator-sdk/scorecard/cmd.go

```diff
@@ -90,11 +90,13 @@ If the argument holds an image tag, it must be present remotely.`,
 		"Disable resource cleanup after tests are run")
 	scorecardCmd.Flags().DurationVarP(&c.waitTime, "wait-time", "w", 30*time.Second,
 		"seconds to wait for tests to complete. Example: 35s")
+	// Use the digest of the latest scorecard-store image
```

Comment:
```suggestion
	// Please note that for Operator-sdk + Preflight + DCI integration in disconnected environments, 
	// it is necessary to refer to storage-image and untar-image using their digests instead of tags. 
	// If you need to make changes to these images, please ensure that you always use the digests.
```